package.json

```diff
@@ -43,5 +43,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "enzyme": "^3.11.0",
+    "jest": "^24.9.0",
+    "react-test-renderer": "^17.0.1",
+    "redux-mock-store": "^1.5.4"
```

Comment:
You added redux-mock-store and enzyme but didn't use it at all. I believe it can be deleted for now.